### ISSUE TITLE
[Update Request]: LINQPad.LINQPad.8

### ISSUE THREAD

**Issue by DevAlsicJanDeclercq** [microsoft/winget-pkgs#166181]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

LINQPad.LINQpad.8

### Package Version

8.5.5

### Please describe the changes you would like to see

There is a new release available (8.5.5)
Url remains the same.  => hash mismatch

**Comment by Dragon1573** [comment]
Is this package still available in this repository? 

```text
21:36:10 C:\...\winget-pkgs  [master ≡] 2.264s 0x0001 pwsh> komac show LINQPad.LINQpad.8
Error: 
   0: LINQPad.LINQpad.8 does not exist in microsoft/winget-pkgs
   1: Failed to retrieve directory content of manifests/l/LINQPad/LINQpad/8

21:36:20 C:\...\winget-pkgs  [master ≡] 1.895s 0x0001 pwsh> winget search -e LINQPad.LINQpad.8
找不到与输入条件匹配的程序包。
```

**Comment by DevAlsicJanDeclercq** [comment]
``` powershell
PS C:\> winget search linqpad
Name      Id                Version Source
-------------------------------------------
LINQPad 5 LINQPad.LINQPad.5 5.48.0  winget
LINQPad 6 LINQPad.LINQPad.6 6.15.12 winget
LINQPad 7 LINQPad.LINQPad.7 7.8.10  winget
LINQPad 8 LINQPad.LINQPad.8 8.4.11  winget
```

**Comment by DevAlsicJanDeclercq** [comment]
> Is this package still available in this repository?
> 
> ```
> 21:36:10 C:\...\winget-pkgs  [master ≡] 2.264s 0x0001 pwsh> komac show LINQPad.LINQpad.8
> Error: 
>    0: LINQPad.LINQpad.8 does not exist in microsoft/winget-pkgs
>    1: Failed to retrieve directory content of manifests/l/LINQPad/LINQpad/8
> 
> 21:36:20 C:\...\winget-pkgs  [master ≡] 1.895s 0x0001 pwsh> winget search -e LINQPad.LINQpad.8
> 找不到与输入条件匹配的程序包。
> ```

Sorry:     LINQPad.LINQ**P**ad.8

**Comment by Dragon1573** [comment]
> [!NOTE]
>
> This issue should be closed as "completed", since #165494 has successfully merged.

CC @DevAlsicJanDeclercq @stephengillie

**Comment by stephengillie** [comment]
> Note
> 
> This issue should be closed as "completed", since #165494 has successfully merged.
> 
> CC @DevAlsicJanDeclercq @stephengillie

Thanks for helping us to keep our issues relevant.
